src/main/java/com/quirko/logic/SimpleBoard.java

```diff
@@ -83,14 +88,30 @@ public boolean rotateLeftBrick() {
         }
     }
 
+    // Inside of if statement's resource: http://www.mediafire.com/file/c5ai9cvbc4aeau6/2048-Final.zip/file
     @Override
     public boolean createNewBrick() {
-        Brick currentBrick = brickGenerator.getBrick();
+        currentBrick = brickGenerator.getBrick();
+        if (currentBrick.getClass().equals(dotBrickObj.getClass())) {
+            audio = AudioHandler.getInstance();
+            audio.load("selection.wav", "select");
+            audio.play("select", 0);
+        }
         brickRotator.setBrick(currentBrick);
         currentOffset = new Point(4, 0);
         return MatrixOperations.intersect(currentGameMatrix, brickRotator.getCurrentShape(), (int) currentOffset.getX(), (int) currentOffset.getY());
     }
 
+    @Override
+    public boolean changeBrick() {
+        if (currentBrick.getClass().equals(dotBrickObj.getClass())) {
+            Brick currentBrick = brickGenerator.changeNextBrick();
+            brickRotator.setBrick(currentBrick);

```

Comment:
Demo sırasında işini kolaylaştırmak için uygulama belli bir flag ile çalıştırıldıktan sonra, oyun sırasında belli bir keye basıldığında bu özel brickin gelmesini sağla. Böylece istediğin zaman demoda bir dotbrick çıkartabilirsin.